build.gradle

```diff
@@ -157,9 +167,9 @@ project(':programming-core') {
 
     dependencies {
         compile(
-                'org.eclipse.jetty:jetty-server:9.2.14.v20151106',
-                'org.eclipse.jetty:jetty-servlet:9.2.14.v20151106',
-                'org.eclipse.jetty:jetty-xml:9.2.14.v20151106',
+                'org.eclipse.jetty:jetty-server:9.2.29.v20191105',
+                'org.eclipse.jetty:jetty-servlet:9.2.29.v20191105',
+                'org.eclipse.jetty:jetty-xml:9.2.29.v20191105',
```

Comment:
For the sake of homogeneity, why don't you use the same version of jetty as in the scheduling project (which is '9.2.24.v20180105').

Reply:
It's WIP. We should be addressing this soon.